pom.xml

```diff
@@ -93,6 +93,7 @@
         <module>test/bundle-shading-tests</module>
         <module>test/crt-unavailable-tests</module>
         <module>test/architecture-tests</module>
+        <module>test/s3-tests</module>
```

Comment:
I think there are a couple of places we need to update for new modules, can we double check the new module checklist?

Reply:
Yep, will do that. Thanks for the reminder.

Reply:
Done